src/Symfony/Component/Form/Extension/Core/Type/TextareaType.php

```diff
@@ -14,9 +14,18 @@
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class TextareaType extends AbstractType
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefault('invalid_message', 'The text area is invalid.');
```

Comment:
i think we can keep the default value here no? same for `text` and other generic types

Reply:
Yes, I've removed a few custom translations from generic types.